test/enhance/test_shift_rgb.py

```diff
@@ -9,43 +9,43 @@
 class TestRGBShift:
 
     def test_rgb_shift_no_shift(self, device, dtype):
-        r_shift, g_shift, b_shift = 0, 0, 0
+        r_shift, g_shift, b_shift = torch.Tensor([0]), torch.Tensor([0]), torch.Tensor([0])
```

Comment:
```suggestion
        r_shift, g_shift, b_shift = torch.tensor([0]), torch.tensor([0]), torch.tensor([0])
```

`torch.tensor` for creating torch instances, `torch.Tensor` is class types.